packages/ibm-products/src/components/TagOverflow/TagOverflow.stories.js

```diff
@@ -218,13 +225,15 @@ export const ManyTags = Template.bind({});
 ManyTags.args = {
   containerWidth: 500,
   items: tags,
+  ...overflowAndModalStrings,
```

Comment:
can we update the name here to highlight that this uses the modal? it took me longer than i'd like to admit to figure out how to get the modal to display, so including it in the name here would be helpful. maybe like `TagsWithOverflowModal` or something

Reply:
Updated story name as you suggested. I have updated other story names as well to make it more meaningful.